code/datums/diseases/advance/symptoms/sneeze.dm

```diff
@@ -46,7 +55,3 @@ Bonus
 		if(1, 2, 3)
 			if(!suppress_warning)
 				M.emote("sniff")
-		else

```

Comment:
Just change this to `else if(A.properties["transmittable"] >= 9)` and revert all other changes to this file.

Reply:
Otherwise people will only sneeze when it first starts processing.